hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1074,6 +1074,67 @@ def hpat_pandas_series_max_impl(self, axis=None, skipna=True, level=None, numeri
     return hpat_pandas_series_max_impl
 
 
+@overload_method(SeriesType, 'mean')
+def hpat_pandas_series_mean(self, axis=None, skipna=None, level=None, numeric_only=None):
+    """
+    Pandas Series method :meth:`pandas.Series.mean` implementation.
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_mean
+
+    Parameters
+    -----------
+    axis: {index (0)}
+        Axis for the function to be applied on.
+        *unsupported*
+    skipna: :obj:`bool`, default True
+        Exclude NA/null values when computing the result.
+    level: :obj:`int` or level name, default None
+        If the axis is a MultiIndex (hierarchical), count along a particular level, collapsing into a scalar.
+        *unsupported*
+    numeric_only: :obj:`bool`, default None
+        Include only float, int, boolean columns.
+        If None, will attempt to use everything, then use only numeric data. Not implemented for Series.
+        *unsupported*
+
+    Returns
+    -------
+    :obj:
+         Return the mean of the values for the requested axis.
+    """
+
+    _func_name = 'Method mean().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given: {}'.format(_func_name, self))
+
+    if not isinstance(self.data.dtype, types.Number):
+        raise TypingError('{} Currently function supports only numeric values. Given data type: {}'.format(_func_name, self.data.dtype))
+
+    if not isinstance(skipna, (types.Omitted, types.Boolean)) and skipna is not None:
+        raise TypingError(
+            '{} The parameter must be a boolean type. Given type skipna: {}'.format(_func_name, skipna))
+
+    if not (isinstance(axis, types.Omitted) or axis is None) \
+            or not (isinstance(level, types.Omitted) or level is None) \
+            or not (isinstance(numeric_only, types.Omitted) or numeric_only is None):
+        raise TypingError(
+            '{} Unsupported parameters. Given axis: {}, level: {}, numeric_only: {}'.format(_func_name, axis, level,
+                                                                                            numeric_only))
+
+    def hpat_pandas_series_mean_impl(self, axis=None, skipna=None, level=None, numeric_only=None):
```

Comment:
```suggestion
    def hpat_pandas_series_mean_impl(self, axis=None, skipna=True, level=None, numeric_only=None):
```

Reply:
@shssf could you please explain in this issue https://github.com/IntelPython/hpat/issues/233 for me and all why we should replace Pandas signature `skipna=None` with our `skipna=True`?
I really do not understand which criteria I should use to create signature for our functions, which default values I should replace and which to remain. My first suggestion was - do like in Pandas.

Reply:
@PokhodenkoSA I would refer to Pandas sources. Please follow signature defined there. Documentation might have a bugs as a regular code. Anyway, by the algo you implemented it is True despite of the fact it None